modules/gz-math/20241112.1/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
```

Comment:
Please remove windows from presubmit checks for all three modules since the gazebo bazel setup currently does not support windows.

Reply:
Done.